src/main/java/webserver/RequestHandler.java

```diff
@@ -35,96 +34,25 @@ public void run() {
             HttpRequest httpRequest = HttpRequest.of(bufferedReader);
 
             DataOutputStream dos = new DataOutputStream(out);
-
-            byte[] body = getBodyFromRequest(httpRequest);
-
-            response200Header(dos, body.length, httpRequest.getHttpRequestPath().getPath());
-            responseBody(dos, body);
+            HttpResponse httpResponse = HttpResponse.responseHeader(httpRequest);
+            responseHeader(dos, httpResponse);
+            responseBody(dos, httpResponse.getBody());
```

Comment:
아래와 같이 writeResponse 메서드 내에서 header, body를 다 응답해주거나 혹은 이 메서드 내에서 위의 두 메서드를 호출하는게 조금 더 깔끔할 것 같아요. 
```suggestion
            writeResponse(dos, httpResponse)
```